crossterm_style/src/color.rs

```diff
@@ -4,13 +4,14 @@
 use std::clone::Clone;
 use std::io;
 
-use super::*;
-use crate::{Color, ITerminalColor};
-
 #[cfg(windows)]
 use crossterm_utils::supports_ansi;
 use crossterm_utils::{impl_display, Command, Result};
 
+use crate::{Color, ITerminalColor};
+
+use super::*;
```

Comment:
Eventually, I like to get rid of those as well, doesn't need to be done in this PR tough

Reply:
Can you elaborate more? Why to remove `crate::...`? Replace `super::*` with specific imports to avoid `*`?

Reply:
Yea, * is unclear to which types your importing and can create confusion

Reply:
Okay, I'll do this. But if you don't mind, in a separate PR, cause this one is mainly about 2018 edition.

Reply:
Yea totally fine